components/common-data-items.js

```diff
@@ -352,7 +376,7 @@ export function BiosampleDataItems({
       </>
       {pooledFrom?.length > 0 && (
         <>
-          <DataItemLabel>Biosample(s) Pooled From</DataItemLabel>
+          <DataItemLabel>Pooled From Sample(s)</DataItemLabel>
```

Comment:
Similar thing for plurals as above. I know it was done with “(s)” before, as well as below.